backup/Makefile

```diff
@@ -0,0 +1,24 @@
+# Go parameters
+BINARY_NAME=postgres-s3-backup
+BUILD_DIR=dist
+
+# PRERELEASE defines semver prerelease tag based on the state of the current git tree.
+# For a clean tree this is simply short commit hash of HEAD, e.g., "4be01eb".
+# Dirty tree has "-dirty" suffix added, e.g., "4be01eb-dirty".
+PRERELEASE ?= $(shell git describe --match= --always --dirty)
+TIMESTAMP ?= $(shell date +%y%m%d%H%M)
+
+# TAG for the operator docker image, e.g., "dev-4be01eb-dirty".
```

Comment:
nit: Not an `operator` here.

Reply:
Thanks, I missed that. I also removed the first few lines that were not used.